packages/toolpad-app/src/toolpad/Apps/index.tsx

```diff
@@ -428,7 +426,11 @@ interface AppOpenButtonProps {
 function AppOpenButton({ app, activeDeployment }: AppOpenButtonProps) {
   const openDisabled = !app || !activeDeployment;
   let openButton = (
-    <Button disabled={!app || !activeDeployment} component="a" href={app ? `deploy/${app.id}` : ''}>
+    <Button
```

Comment:
Can this one be a `Link` too as it seems to be an internal link?

Reply:
No, it's not under the same `react-router`.
We're mixing next router and react-router. not 100% ideal, but (at the time of writing) next router wasn't really helpful in building the editor with its deep links to subeditors

Reply:
ah ok, nevermind then